script/svg/__init__.js

```diff
@@ -4,64 +4,59 @@ __setupPackage__(__extension__);
  * the SVG parser.
  */
 
-function isNumber(n) {
+var isNumber = function(n) {
   return !isNaN(parseFloat(n)) && isFinite(n);
-}
-
-
-
-(function(){
+};
 
+(function() {
     svg.templateHooks = {};
 
     svg.regexpQuote = function (str, delimiter) {
         return String(str).replace(
             new RegExp(
-                '[.\\\\+*?\\[\\^\\]$(){}=!<>|:\\' + (delimiter || '') + '-]',
-                'g'
+                "[.\\\\+*?\\[\\^\\]$(){}=!<>|:\\" + (delimiter || "") + "-]",
+                "g"
             ),
-            '\\$&'
+            "\\$&"
         );
-    }
+    };
 
     svg.getHooksPattern = function(){
-        var hookNames = [],
-            that = this;
-        for( var i in this.templateHooks )
+        var hookNames = [];
+        for (var i in this.templateHooks)
             hookNames.push(i);
-        
+
         // hook_name( arg1 [, arg2]... )
-        if( hookNames.length ){
-            var pattern = "("+hookNames.join('|')+")\\(([^\\(\\)]+)\\)\\s*;?";
+        if (hookNames.length){
+            var pattern = "("+hookNames.join("|")+")\\(([^\\(\\)]+)\\)\\s*;?";
             return pattern;
         }
-    }
+    };
 
     var global = this;
-    svg.templateHooks.variable = function( varName ){
-        if( varName in global ){
+    svg.templateHooks.variable = function(varName){
+        if (varName in global){
             return global[varName];
         }
-        return '';
-    }
+        return "";
+    };
+
+    svg.templateHooks.prop = function(propName, varName){
```

Comment:
Doesn't the new config detect and point out the missing space between adjacent braces?

Reply:
No, I forgot to add that rule:
https://eslint.org/docs/rules/space-before-blocks

I'll make a new PR and add it.

Reply:
Here it is: #2457